client/app/bundles/course/assessment/pages/AssessmentShow/QuestionsManager.tsx

```diff
@@ -66,14 +68,17 @@ const QuestionsManager = (props: QuestionsManagerProps): JSX.Element => {
     if (sourceIndex === destinationIndex) return;
 
     moveItemAndUpdate(sourceIndex, destinationIndex);
+    props.setSyncStatus('Syncing');
```

Comment:
What happens if there is an error in CM backend when user attempts to `moveItemAndUpdate` and it fails (CM side)?

Reply:
in this implementation, then the synchronizing will still take place and since the ordering does not change, then the order will persist (in other words, nothing will happen other than calling one more unnecessary API to backend)

I have revised this part by only invoking state change only when the `moveItemAndUpdate` is successful.